app/i18n/index.js

```diff
@@ -28,6 +28,11 @@ function loadTranslation(locale) {
             localeData = require('react-intl/locale-data/de');
             momentData = require('moment/locale/de');
             break;
+        case 'en-AU':
```

Comment:
I guess this should still be removed.

Reply:
I started in the wrong branch on my local machine.  Removed en-AU